CNAME

```diff
@@ -1 +0,0 @@
-read.wiki.org
```

Comment:
What's the process you'd expect for update `gh-pages` then? When I tested merging your branch to `gh-pages` it deleted the `CNAME` file rather than leaving it untouched.

Seems like that would require manually tweaking the `gh-pages` branch each time we pulled updates over to? Or is there some other trick I'm missing here.